report/report.tex

```diff
@@ -239,12 +236,12 @@ \subsubsection{Semantic Checking (Janus-side)}
 \begin{minipage}{.5\textwidth}
 \codeErr{semJ}
 \end{minipage}
-    
-
+\vspace*{1cm}
 \subsection{Examples of \textit{Hanus}}
 \code{Division}{divide}
-The division example is a simple example of what \textit{Hanus} is capable of. This function computes the division and the remainder. When the variable is called in reverse, it can be used to compute the multiplication. Because the \textit{main} procedure calls divide and then uncalls divide with the same arguments, main acts as the identity function for most parameters. With this program, it is possible to explain the loop in \textit{Hanus}. The loop has a precondition and a postcondition and a body. The precondition has to be $True$ only the first iteration and the postcondition has to be $True$ only the last iteration. When the loop is reversed, precondition and postcondition change places. 
-    
+The division example is a simple example of what \textit{Hanus} is capable of. This function computes the division and the remainder. When the function is called in reverse, it can be used as multiplication. Because the \inlinecode{main} procedure calls divide and then uncalls divide with the same arguments, main acts as the identity function for most parameters. With this program, it is possible to explain the loop in \textit{Hanus}. The loop has a precondition and a postcondition and a body. The precondition has to be \inlinecode{True} only the first iteration and the postcondition has to be \inlinecode{True} only the last iteration. When the loop is reversed, precondition and postcondition change places. 
```

Comment:
"main acts as the identity function for most parameters" doesn't it simply act as the identity function for all possible (valid) parameters? Otherwise we've broken the entire concept of Hanus right?